src/gui/PatternsManagerWindow.ui

```diff
@@ -14,10 +14,22 @@
    <string>Patterns manager</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
-   <item row="0" column="0" alignment="Qt::AlignHCenter">
-    <widget class="QLabel" name="label">
-     <property name="text">
-      <string>Pattern</string>
+   <item row="0" column="0" colspan="2">
+    <widget class="QListWidget" name="patterns">
+     <property name="cursor" stdset="0">
+      <cursorShape>WhatsThisCursor</cursorShape>
+     </property>
+     <property name="toolTip">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Patterns currently configured for the active profile.&lt;/p&gt;&lt;p&gt;Regex patterns are distinguished by being &lt;span style=&quot; font-weight:700;&quot;&gt;bold&lt;/span&gt;.&lt;/p&gt;&lt;p&gt;You can change the pattern by &lt;span style=&quot; font-weight:700;&quot;&gt;double-clicking&lt;/span&gt; on it and editing the text.&lt;/p&gt;&lt;p&gt;Note: The type of the pattern (glob/regex) is set on creation and cannot be changed.&lt;/p&gt;&lt;p&gt;You can reorder patterns by &lt;span style=&quot; font-weight:700;&quot;&gt;drag-and-droping&lt;/span&gt; them around in the list.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Not sure if a tooltip is the right place for this
Maybe a popup?
We really ought to have proper docs somewhere

Reply:
Hmm, I agree, but I don't really know a better way how to handle this at this point. 🤔 

I have some early feedback and patterns are confusing. It is hard to understand that the settings in the bottom part are tied to pattern selected in the above part of the window and won't apply universally.

I agree that docs would be nice. I was also thinking an interactive tutorial would be nice, just to point the user to what each thing does (something similar that Mod Organizer 2 can do, for example when you try to install a mod with a weird directory structure).

Reply:
The interactive tutorial is a good idea, but I have absolutely no idea how to do it. I guess we can learn

> I have some early feedback and patterns are confusing. It is hard to understand that the settings in the bottom part are tied to pattern selected in the above part of the window and won't apply universally.

If you want to discuss it, please open an issue

---

In the meantime, let's keep this tooltip

Reply:
Issue created.